app/Http/Controllers/CourseController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Course;
+use Illuminate\Contracts\View\View;
+use Illuminate\Support\Facades\Auth;
+
+class CourseController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function index(): View
+    {
+        return view('courses.index', [
+            'courses' => Course::all(),
+        ]);
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Course  $course
+     * @return \Illuminate\View\View
+     */
+    public function show(Course $course)
```

Comment:
Add a return type hint, e.g.

```suggestion
    public function show(Course $course): View
```